lib/core/src/lib/form/components/widgets/core/form-field-validator.ts

```diff
@@ -583,3 +584,39 @@ export const FORM_FIELD_VALIDATORS = [
     new MinDateTimeFieldValidator(),
     new MaxDateTimeFieldValidator()
 ];
+
+export const momentToDateFnsMap = {
+    M: 'M',
+    D: 'd',
+    Y: 'y',
+    A: 'a'
+};
+
+export const dateFnsToMomentMap = {
+    M: 'M',
+    d: 'D',
+    y: 'Y',
+    a: 'A'
+};
+
+export function convertMomentToDateFnsFormat(dateDisplayFormat: string): string {
+    for (const [search, replace] of Object.entries(momentToDateFnsMap)) {
+        dateDisplayFormat = dateDisplayFormat.replace(new RegExp(search, 'g'), replace);
+    }
+    return dateDisplayFormat;
+}
+
+export function convertDateFnsToMomentFormat(dateDisplayFormat: string): string {
+    for (const [search, replace] of Object.entries(dateFnsToMomentMap)) {
+        dateDisplayFormat = dateDisplayFormat.replace(new RegExp(search, 'g'), replace);
+    }
+    return dateDisplayFormat;
+}
+
+export function formatUpdate(date: number|Date, dateFormat: string): string {
```

Comment:
move to DateFnsUtils class and use better naming, like `formatDate`, it is already implied that some update is going to happen